riscv-c-api.md

```diff
@@ -94,6 +97,15 @@ The value of `__riscv_v_elen_fp` is defined by the following rules:
 If multiple rules apply, the maximum value is taken.
 If none of the rules apply, `__riscv_v_elen_fp` is undefined.
 
+### __riscv_misaligned_{fast,slow,bad}
+
+These can be used in common library code to compile time segregate code which relies
+on misaligned access being fast or not.
+A typical complier could (but not necessarily) map fast variant to -mno-strict-align
+and bad to -mstrict-align, if specified.
```

Comment:
I would like to drop that since clang has different behavior here, it just accept and ignore `-mno-strict-align` and `-mstrict-align` for RISC-V clang.

Or move it become a note to say GCC is using that way to decide which macro should be defined.

Reply:
Sorry not clear what you mean: clang ignores either of them ? and uses a different toggle to affect misaligned access codegen or doesn't support that at all (extremely unlikely)

Reply:
> Sorry not clear what you mean: clang ignores either of them ? and uses a different toggle to affect misaligned access codegen or doesn't support that at all (extremely unlikely)

Yes, RISC-V clang accept that and then ignore that for now - clang did support misaligned code gen and some more are coming up[1], option handling in clang part can easily be implement/fix later.

Anyway just second thought about this: This should not major blocker issue here, just a minor implementation issue, and you already mention `(but not necessarily)`.